src/pnpm/doPnpmBumpVersion.ts

```diff
@@ -0,0 +1,22 @@
+import { runOnChanged } from "./utils/runOnChanged";
+
+/**
+ * Bump all changed packages to version {newVersion}
+ * @param newVersion the version to bump can be a version number of bump type
```

Comment:
`of` or `or` ? :see_no_evil:
`can be a version number of bump type` is not clear for me